src/lib/sections/Navigation/Icon/Icon.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+
+import { Navigation } from "@/lib/sections/Navigation/Navigation";
+
+it("renders the light version of the icon by defaults", () => {
+  render(
+    <span data-testid="test-icon-wrapper">
+      <Navigation.Icon name="information" />
+    </span>
+  )
+  expect(screen.getByTestId("test-icon-wrapper").firstChild).toHaveClass("p-icon--information is-light")
+})
```

Comment:
In this case it might be better to use `container.firstChild` instead. (or use querySelector to target the `i` element). While these techniques are generally discouraged, it makes sense to use them in such low level tests in a component library.

Reply:
Another option is to use a testid on the icon component directly.

Reply:
Decided to go with the `container.firstChild` solution for this.